numpy/__init__.py

```diff
@@ -208,7 +208,7 @@
         # https://www.python.org/dev/peps/pep-0562/
         def __getattr__(attr):
             if attr == 'testing':
-                import numpy.testing as testing
+                from . import testing
```

Comment:
This seems to be breaking tests

Reply:
Strange, It works locally.
I'll roll back that line and see what happens.

Reply:
python 3.8 only